File: api_gateway/src/main/java/com/ticketflow/api_gateway/models/client_models/user_api/ProfileClientModel.java
package com.ticketflow.api_gateway.models.client_models.user_api;

import java.time.LocalDate;

public class ProfileClientModel {
    private Long phoneNumber;
    private LocalDate birthday;

    public ProfileClientModel(Long phoneNumber, LocalDate birthday) {
        this.phoneNumber = phoneNumber;
        this.birthday = birthday;
    }

    public Long getPhoneNumber() {
        return phoneNumber;
    }

    public LocalDate getBirthday() {
        return birthday;
    }
}

File: backend/dotnet/ticketflow/TicketFlow.ApiGateway/Models/ApiGateway/UserApi/ProfileClientModel.cs
using System;

namespace TicketFlow.ApiGateway.Models.ApiGateway.UserApi
{
    public class ProfileClientModel
    {
        public ProfileClientModel(long phoneNumber, DateTime birthday)
        {
            PhoneNumber = phoneNumber;
            Birthday = birthday;
        }

        public long PhoneNumber { get; }

        public DateTime Birthday { get; }
    }
}